blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ActionProxiesImpl.java

```diff
@@ -86,13 +91,17 @@ public Link getLink() {
             @Override
             public boolean apply(Action action) {
                 return action != null
-                    && (action.getClass().getAnnotation(ExportedBean.class) != null || alwaysAllowAction.apply(action))
-                    && !BANNED_ACTIONS.contains(action.getClass().getName());
+                    && (action.getClass().getAnnotation(ExportedBean.class) != null || alwaysAllowAction.apply(action));
```

Comment:
No, we don't need it since actions are not returned by default. When we will have actions thru extensions then they will pick what they want. See https://issues.jenkins-ci.org/browse/JENKINS-43742